packages/cli/src/Logger.ts

```diff
@@ -112,21 +109,17 @@ export class Logger implements ILogger {
 		this.log('error', message, meta);
 	}
 
-	verbose(message: string, meta: object = {}): void {
-		this.log('verbose', message, meta);
-	}
-
 	warn(message: string, meta: object = {}): void {
 		this.log('warn', message, meta);
 	}
-}
-
-let activeLoggerInstance: Logger | undefined;
 
-export function getLogger() {
-	if (activeLoggerInstance === undefined) {
-		activeLoggerInstance = new Logger();
+	/** @deprecated: use `debug` directly from `n8n-workflow */
+	debug(message: string, meta: object = {}): void {
+		LoggerProxy.debug(message, meta);
 	}
 
-	return activeLoggerInstance;
+	/** @deprecated: use `debug` directly from `n8n-workflow */
+	verbose(message: string, meta: object = {}): void {
```

Comment:
If we're merging `verbose` into `debug`, should we remove `verbose` from `verbose` at `schema.ts`?

Reply:
I think I'm going to revert this change. This is also from before https://github.com/n8n-io/n8n/pull/7479